Source/BoundaryConditions/WarpXEvolvePML.cpp

```diff
@@ -57,7 +58,22 @@ WarpX::DampPML (int lev, PatchType patch_type)
     if (!do_pml) return;
 
     WARPX_PROFILE("WarpX::DampPML()");
+#ifdef WARPX_DIM_RZ
+    amrex::ignore_unused(patch_type);
+    if (pml_rz[lev]) {
+        pml_rz[lev]->ApplyDamping({ Efield_fp[lev][0].get(), Efield_fp[lev][1].get(), Efield_fp[lev][2].get() },
+                                  { Bfield_fp[lev][0].get(), Bfield_fp[lev][1].get(), Bfield_fp[lev][2].get() },
+                                  dt[lev]);
+    }
+#endif
+    if (pml[lev]) {
+        DampPML_Cartesian (lev, patch_type);
+    }
+}
 
+void
+WarpX::DampPML_Cartesian (int lev, PatchType patch_type)
```

Comment:
```suggestion
WarpX::DampPML_Cartesian (const int lev, PatchType patch_type)
```

Reply:
I can make the change, but this is how it was originally, without the `const`. I'll make the same change to the `DampPML` routines above which have the same issue.